drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -33,11 +33,14 @@
 #include "project_settings.h"
 #include "rasterizer_canvas_gles3.h"
 #include "servers/visual/visual_server_raster.h"
+#include "core/math/math_funcs.h"
 
 #ifndef GLES_OVER_GL
 #define glClearDepth glClearDepthf
 #endif
 
+
```

Comment:
Please don't add extraneous whitespace, this does not follow the style guide: http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html